README.mdown

```diff
@@ -1,131 +1,211 @@
-# Generate Dependent Styles
-The script `generate_styles.rb` can be used to automatically generate dependent CSL styles for publishers that use the same citation format(s) across multiple journals. All that is needed is a collection of journal metadata (journal title, ISSN, eISSN, citation format, etc.).
+#### Notice
+This repository was created on 2016-03-10 as a dedicated home for the journal metadata previously stored at https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles (in the now-removed "generate_dependent_styles" subdirectory).
 
-## Preparing journal metadata
-Each publisher gets its own metadata directory in the `generate_dependent_styles` subdirectory of this repository (e.g. `asm` for the American Society for Microbiology). The file [publishers.json](https://github.com/citation-style-language/utilities/blob/master/generate_dependent_styles/publishers.json) matches directories to publisher names.
+# Journals
+## Metadata repository for dependent CSL styles
+While there seems to be an endless variety of citation formats, many publishers fortunately standardize on one, or at most a few, citation formats for their collection of journals.
+This reduces the number of "independent" CSL citation styles we need to maintain, and allows us to create lightweight "dependent" CSL styles for the individual journals.
```

Comment:
But, but, ... it's a long sentence and I'd like to give the reader a place to catch their breath. Is it because the two sentence parts aren't independent clauses? (and that if I would want to use the comma, I'd have to write: "This reduces ..., and **it** allows"?)